src/main/initializers/init-ipc-main-handlers/init-ipc-main-handlers.ts

```diff
@@ -14,25 +14,23 @@ import { catalogEntityRegistry } from "../../catalog";
 import { pushCatalogToRenderer } from "../../catalog-pusher";
 import { ClusterManager } from "../../cluster-manager";
 import { ResourceApplier } from "../../resource-applier";
-import { WindowManager } from "../../window-manager";
-import path from "path";
 import { remove } from "fs-extra";
-import { getAppMenu } from "../../menu/menu";
-import type { MenuRegistration } from "../../menu/menu-registration";
-import type { IComputedValue } from "mobx";
 import { onLocationChange, handleWindowAction } from "../../ipc/window";
 import { openFilePickingDialogChannel } from "../../../common/ipc/dialog";
 import { showOpenDialog } from "../../ipc/dialog";
 import { windowActionHandleChannel, windowLocationChangedChannel, windowOpenAppMenuAsContextMenuChannel } from "../../../common/ipc/window";
 import { getNativeColorTheme } from "../../native-theme";
 import { getNativeThemeChannel } from "../../../common/ipc/native-theme";
+import type { MenuItemsOpts } from "../../menu/get-app-menu-items.injectable";
+import type { GetAbsolutePath } from "../../../common/path/get-absolute-path.injectable";
 
 interface Dependencies {
-  electronMenuItems: IComputedValue<MenuRegistration[]>;
+  getAppMenuItems: () => MenuItemsOpts[];
```

Comment:
Shouldn't this be `IComputedValue<MenuItemsOpts[]>`?

Reply:
That's better. Changed.